docker-build

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+# This file is a part of IoT-LAB gateway_code
+# Copyright (C) 2015 INRIA (Contact: admin@iot-lab.info)
+# Contributor(s) : see AUTHORS file
+#
+# This software is governed by the CeCILL license under French law
+# and abiding by the rules of distribution of free software.  You can  use,
+# modify and/ or redistribute the software under the terms of the CeCILL
+# license as circulated by CEA, CNRS and INRIA at the following URL
+# http://www.cecill.info.
+#
+# As a counterpart to the access to the source code and  rights to copy,
+# modify and redistribute granted by the license, users are provided only
+# with a limited warranty  and the software's author,  the holder of the
+# economic rights,  and the successive licensors  have only  limited
+# liability.
+#
+# The fact that you are presently reading this means that you have had
+# knowledge of the CeCILL license and that you accept its terms.
+
+import os
+
+os.system('docker build -t fitiotlab/iot-lab-gateway .')
```

Comment:
Do we really need another command line wrapper just for this ? It's be enough to mention it in the README.

Reply:
Well, in that case it's important the the built image is named fitiotlab/iot-lab-gateway, so that it can be used by docker-run, 

```./docker-build``` instead of     ```docker build -t fitiotlab/iot-lab-gateway .```, it's shorter and we are lazy.

Though it could be cool to know the docker build command, because ```docker build -t fitiotlab/iot-lab-gateway tests``` builds a slightly different image in which tests can be run, for example:

    docker build -t fitiotlab/iot-lab-gateway tests
    ./docker-run tox